site/app/views/admin/ExtensionsView.php

```diff
@@ -31,17 +34,20 @@ public function displayExtensions($gradeables) {
             $current_exceptions[] = ['user_id' => $user->getId(),
                                           'user_givenname' => $user->getDisplayedGivenName(),
                                           'user_familyname' => $user->getDisplayedFamilyName(),
-                                          'late_day_exceptions' => $user->getLateDayExceptions()];
+                                          'late_day_exceptions' => $user->getLateDayExceptions(),
+                                          'reason_for_exception' => $user->getReasonForException()];
         }
         if (empty($current_exceptions)) {
             $current_exceptions = null;
         }
+        $reasons = ["illness", "interview", "travel", "personal issue"];
```

Comment:
Why can't you just inline this variable?

Reply:
Could you explain what that means please?

Reply:
`$reasons` is only used in one location after being defined here.  You could eliminate the temporary variable by removing this line and changing the block of code below this to something like the following:

```
return $this->core->getOutput()->renderTwigTemplate("admin/Extensions.twig", [
             "gradeables" => $gradeables,
             "student_full" => $student_full,
             "current_gradeable" => $current_gradeable,
             "current_exceptions" => $current_exceptions,
             "reasons" => [
                 "illness",
                 "interview",
                 "travel",
                 "personal issue",
             ],
             "csrf_token" => $this->core->getCsrfToken(),
         ]);
```

This is just a suggestion though, so feel free to disregard if you disagree though.